bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/dto/onecall/Daily.java

```diff
@@ -33,6 +33,9 @@ public class Daily {
     private int dt;
     private int sunrise;
     private int sunset;
+    private int moonrise;
+    private int moonset;
+    private double moon_phase;
```

Comment:
To follow the Java naming conventions, rename that field and add an annotation:

```suggestion
    @SerializedName("moon_phase")
    private double moonPhase;
```